.devcontainer/devcontainer.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Why delete these codes in your commit? It should be here because it's part of local development enverionment.